nss-ivory-trolls/WorkforceManagement
# Ac employee controller list & detail views

# Description
Added functional list and details views to the employee controller. Updated ReadMe.

## Type of change
Please delete options that are not relevant.

# Testing Instructions

- Git fetch -all
- Git checkout ac-employeecontroller
- Dotnet run
- Click on 'Employees' link in navigation bar
- Confirm that a list of employees populates. Each employee entry should include the following:
1. First Name
2. Last Name
3. Department Name
- Click on an individual employee within the employee list
- Confirm that you are re-directed to the details view of an individual employee. This view should include the following information:
1. First Name
2. Last Name
3. Department Name
4. Computer Make
5. A List of training programs that the employee has attended or plans to attend

# Checklist:
